### PR TITLE
SAK-44561 Error parsing the date/time PA SYSTEM Create Banner

### PR DESCRIPTION
**Pull request by MRutea**

### CODE REVIEW

File: pasystem/pasystem-tool/tool/src/webapp/scripts/pasystem-tool.js [hunk 1]
```diff
@@ -23,7 +23,6 @@ $(function () {
             // add clear action if present
             $datepicker.siblings().find(".clear-datepicker-btn").click(function() {
               $datepicker.val("");
-              $("[name='" + $datepicker.attr('id') + "_selected_datetime']").val("");
```
Comment (adrianfish) [inline pasystem/pasystem-tool/tool/src/webapp/scripts/pasystem-tool.js:26]: I wouldn't just remove this, unless you know exactly where it's used. It looks like it's the hidden field generated by the date picker. I'd play it safe and do:
const hidden = this.querySelector(`[name='${$datepicker.attr('id')}_selected_datetime']`);
hidden && hidden.value = "";

**Review by adrianfish — CHANGES_REQUESTED** [review]
See what you think about my comment.

**Comment by MRutea** [comment]
@adrianfish 
Hi, Adrian;

Thanks for your comment.
I think the line was no longer used, after the different tests I did.
That's why I removed it.

But you're right.
Let's play safe. I added the code you wrote for me.

Thanks again.

**Review by adrianfish — APPROVED** [review]
Good to go, as long as you tested my suggested change - just in case :)

**Pull request merged**